docs/browse/script.js

```diff
@@ -130,13 +127,13 @@ function renderActiveEINs() {
   viewModel.getShowList().forEach((ein) => {
     const name = Charity.getCharity(ein)?.name || "???";
     const $tag = $(
-      '<div class="filter-tag flex items-center gap-0.5 rounded border border-green bg-green/10 text-green rounded-md px-2 py-1 text-xs"></div>'
+      '<div class="filter-tag flex items-center gap-0.5 rounded border border-green bg-green/10 text-green px-2 py-1 text-xs"></div>'
     );
-    const $text = $(`<span title="${name}"></span>`).text(
-      ein.slice(0, 2) + "-" + ein.slice(2)
+    const $text = $(`<span title="EIN: ${ein.split(':')[0].slice(0, 2)}-${ein.split(':')[0].slice(2)}"></span>`).text(
```

Comment:
Good change, should go in J's tool too! Extra credit: a hover with the c.longEIN?